### ISSUE TITLE
Translate LLVM functions to symbolic form lazily

### ISSUE THREAD

**Issue by atomb** [GaloisInc/llvm-verifier#7]
The current `Codebase` implementation translates an entire LLVM bitcode file into symbolic form immediately. Sometimes we want to analyze only some small subset of the functions in a bitcode file, and others use features not supported by the symbolic translation.

**Comment by ntc2** [comment]
Would this solve the problem where unreachable floating point code makes `lss` fail with
```
lss: user error (Floating point operations not supported.)
```
?

**Comment by atomb** [comment]
Yes, it would solve that problem if the floating point instructions are in functions that are never called from the code you're analyzing.

**Comment by ntc2** [comment]
Related: https://github.com/GaloisInc/saw-script/issues/154.
